Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml

```diff
@@ -286,6 +286,7 @@
                                       OpticalMarginAlignment="TrimSideBearings" />
                     <!--  Use grid to toggle visibility based on IsClosable property and inner border for hover animations.  -->
                     <Border x:Name="CloseButtonContainer"
+                            x:Uid="/Microsoft.Toolkit.Uwp.UI.Controls/Resources/WindowsCommunityToolkit_TabView_CloseButton"
```

Comment:
I think the button is getting focused for Narrator, so I think this needs to be on the button vs. the border, no?